xwiki-platform-core/xwiki-platform-rendering/xwiki-platform-rendering-macros/xwiki-platform-rendering-macro-message/src/main/java/org/xwiki/rendering/internal/macro/message/MacroIconPrettyNameProvider.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.rendering.internal.macro.message;
+
+import org.xwiki.component.annotation.Component;
+import org.xwiki.localization.ContextualLocalizationManager;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+/**
+ * XWiki specific component to provide an alternative text for an icon.
+ *
+ * @version $Id$
+ * @since 16.7.0RC1
+ */
+@Component(roles = MacroIconPrettyNameProvider.class)
+@Singleton
+public class MacroIconPrettyNameProvider
```

Comment:
This does not override anything, it should have extended the xwiki-rendering `MacroIconPrettyNameProvider` (and be named `XWikiMacroIconPrettyNameProvider` to be more clear).

Reply:
Fixed in https://github.com/xwiki/xwiki-platform/commit/eb30e7074a52d8d60b2da00459d6a36b263a4607 :+1: Thank you